Foundations/Pages/Relationships.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  Relationships.swift
+//  Foundations
+//
+//  Created by José Danilo Silva do Carmo on 23/03/23.
+//
+
+import SwiftUI
+
+struct Relationships: View {
+    let profile: MyProfile = MyProfile()
+
+    let relationships: [Relationship] = [
+        Relationship(name: "Kiev Gama", image: "kiev-gama"),
+        Relationship(name: "Lisa Mello", image: "lisa-mello")
+    ]
+
+    var body: some View {
+        NavigationStack {
+            VStack {
+                Text("Veja as pessoas de sua equipe que você já trabalhou e quais ainda precisa conhecer melhor")
+                    .font(.custom("Inter-Light", size: 16))
+                    .foregroundColor(.gray)
+                ForEach(relationships, id: \.self, content: {
+                    relationship in
+                    NavigationLink(value: relationship) {
+                        ProfileCircle(imagePath: relationship.image,
+                                      profileName: relationship.name)
+                    }
+                })
+                NavigationLink(value: profile) {
+                    ProfileCircle(imagePath: "profile",
+                                  profileName: "Você")
+                }
+            }
+            .navigationTitle("Minhas conexões")
+            .navigationBarTitleDisplayMode(.inline
+            )
+            .toolbar {
+                ToolbarItem(placement: .principal) {
+                    Text("Minhas conexões").font(.title2)
+                        .bold()
+                }
+            }
+            .navigationDestination(for: Relationship.self){ relationship in
+                SomeView(texto: relationship.name)
```

Comment:
Here you will add the view to be rendered on navigation click. Perceive that there're two destinations because are two links